lib/between_meals/changes/cookbook.rb

```diff
@@ -44,19 +38,59 @@ def self.explode_path(path, cookbook_dirs)
         nil
       end
 
+      def self.symlinked_dir?(link_path)
+        File.symlink?(link_path) &&
+          File.directory?(File.absolute_path(link_path))
+      end
+
+      def self.map_symlinks(files)
+        # For each symlink get the source path, if any files have changed under
```

Comment:
Actually this block just creates a hash of *every* symlink in the entire repo. 

Interestingly, do we want to be recursive? I don't think we do, I think we only want directories directly under the cookbook dir.

Reply:
Yes, This is what we want. As written this will keep two cookbooks in sync, or any file within two cookbooks. 

example one is link:
link chef/itchef/coookbooks/core/foo
source chef/coookbooks/core/foo

example 2:
link chef/itchef/coookbooks/core/foo/files/default/chefctl.rb
source chef/coookbooks/core/foo/files/default/chefctl.rb

Reply:
OHhhh that seems SUPER bad to me. We've seen a lot of problems with symlinks into cookbooks... cookbooks are logical units, we should only support entire cookbooks.